web/crux-ui/src/validations/registry.ts

```diff
@@ -16,7 +16,7 @@ const shouldResetMetaData = { reset: true }
 const registryCredentialRole = yup
   .mixed()
   .meta(shouldResetMetaData)
-  .when(['type', '_private'], {
+  .when(['type', 'private'], {
```

Comment:
imo you should also fix the argument few lines below.

Also `registrySchema ` contains a `_private` shouldn't that be fixed too?

Reply:
'private' is a reserved word in strict mode. Everywhere else we can use it without '_'